tests/QueueableActionTest.php

```diff
@@ -47,6 +50,12 @@
     Queue::assertPushed(ActionJob::class);
 });
 
+test('an action can be queued and receives job property', function () {
+    $action = new ActionReturningJob();
+    $job = $action->onQueue()->execute();
+    assertInstanceOf(\Illuminate\Foundation\Bus\PendingDispatch::class, $job);
```

Comment:
Could you use `Pest`s expectations here?

https://pestphp.com/docs/expectations#expect-toBeInstanceOf

Reply:
ok, thanks!